update-runtime-jdk8-version.sh

```
@@ -8,5 +8,8 @@ if [ -z "$DEST_VERSION" ]; then
   exit 1
 fi
 
-# git grep -rn '2.2.12' | cat | awk -F: '{print $1}' | xargs -I {} sed -i '' 's/2.2.12/2.2.13/g' {}
-grep -rn --include="*.xml" "<koupleless.runtime.version>1\..*</koupleless.runtime.version>" | cat | awk -F: '{print $1}' | xargs -I {} sed -i '' "s/\<koupleless.runtime.version\>1\..*\<\/koupleless.runtime.version\>/\<koupleless.runtime.version\>$1\<\/koupleless.runtime.version\>/g" {}
+if [[ "$(uname)" == "Darwin" ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)